### ISSUE TITLE
Templates don't like top-level lists

### ISSUE THREAD

**Issue by marwatk** [pastdev/clconf#40]
Command-level getv works fine with a top-level list:

```
$ clconf --pipe getv /0 <<'EOF'
[{"key": "value"}]
EOF
key: value
```

But a template is not:

```
$ clconf --pipe getv / --template-string '{{ getv "/0" }}' <<'EOF'
[{"key": "value"}]
EOF
Error: template execute: execute template: template: cli:1:3: executing "cli" at <getv "/0">: error calling getv: /0: key does not exist
```

**Comment by marwatk** [comment]
But it's very specifically only root's that have subkeys:
```
$ clconf --pipe getv / --template-string '{{ getv "/key" }}' <<EOF
key: val
EOF
val
```

**Comment by marwatk** [comment]
This is because the template is operating on a key-value store, and has no concept of keys that don't have direct values. We kinda [call this out in the docs](https://github.com/pastdev/clconf#getv-templates), but the implications are definitely not obvious.

**Comment by marwatk** [comment]
Fixed with #14.
